problem_details.py

```diff
@@ -8,32 +8,38 @@
       _("You must provide the URL to an Authentication For OPDS document to register a library."),
 )
 
-INVALID_AUTH_DOCUMENT = pd(
-      "http://librarysimplified.org/terms/problem/invalid-opds-auth-document",
+INVALID_INTEGRATION_DOCUMENT = pd(
+      "http://librarysimplified.org/terms/problem/invalid-integration-document",
       400,
-      _("Invalid Authentication For OPDS document"),
-      _("The submitted URL did not return a valid Authentication For OPDS document."),
+      _("Invalid Integration document"),
 )
 
-AUTH_DOCUMENT_TIMEOUT = pd(
+TIMEOUT = pd(
       "http://librarysimplified.org/terms/problem/timeout",
       408,
       _("Request timed out"),
       _("Attempt to retrieve an Authentication For OPDS document timed out."),
 )
 
 
-AUTH_DOCUMENT_NOT_FOUND = pd(
-    "http://librarysimplified.org/terms/problem/auth-document-not-found",
+INTEGRATION_DOCUMENT_NOT_FOUND = pd(
+    "http://librarysimplified.org/terms/problem/integration-document-not-found",
     400,
-    title=_("Authentication document not found"),
-    detail=_("No OPDS authentication document was present at the specified URL."),
+    title=_("Document not found"),
+)
+
+AUTH_DOCUMENT_TIMEOUT = pd(
+      "http://librarysimplified.org/terms/problem/timeout",
+      408,
+      _("Request timed out"),
+      _("Attempt to retrieve the OPDS root feed timed out."),
```

Comment:
I don't think you're using this problem detail (and if you are, its name doesn't match its description).